libraries/appnexusKeywords/anKeywords.js

```diff
@@ -60,24 +60,10 @@ export function convertKeywordStringToANMap(keyStr) {
 function convertKeywordsToANMap(kwarray) {
   const result = {};
   kwarray.forEach(kw => {
-    // if = exists, then split
-    if (kw.indexOf('=') !== -1) {
-      let kwPair = kw.split('=');
-      let key = kwPair[0];
-      let val = kwPair[1];
-
-      // then check for existing key in result > if so add value to the array > if not, add new key and create value array
-      if (result.hasOwnProperty(key)) {
-        result[key].push(val);
-      } else {
-        result[key] = [val];
-      }
-    } else {
-      if (!result.hasOwnProperty(kw)) {
-        result[kw] = [];
-      }
```

Comment:
To note - I am still unsure about removing this part of the functionality as it was previously in our adapter code, so it could have been used by someone for their setup and it would then stop sending the data properly to the endpoint.  

Since it seemingly was approved by others in the community, I'm just mentioning this thought; it is not a blocker for the PR's acceptance.